### PR TITLE
Support URLNet in accuracy checker

### PR DESCRIPTION
**Pull request by pwolnows**
The current implementation supports URLNet with 2 inputs. 
There is no tokenization of input_1 data, instead input_1 is fed with tokenized data from input_file defined in URLNet dataset.
Tokenization of input_1 requires TADK Tokenizer that is not supported on all validation platforms.

### CODE REVIEW

**Review by sstrehlk — APPROVED** [review]

File: tools/accuracy_checker/accuracy_checker/metrics/classification.py [hunk 1]
```diff
@@ -671,3 +671,56 @@ def reset(self):
     @classmethod
     def get_common_meta(cls):
         return {'target': 'higher-worse'}
+
+
+class URLNetAccuracy(Metric):
+    __provider__ = 'urlnet_score'
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/classification.py:677]: could you please provide more clear metric name, this approach is not related to specific model

Comment (pwolnows) [reply tools/accuracy_checker/accuracy_checker/metrics/classification.py:677]: renamed to class to UrlCassificationScore and __provider__ to url_classification_score

**Review by eaidova — APPROVED** [review]
